practice-two/src/components/PircesFilterPopover.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect, useRef } from 'react';
+
+// Components
+import Button from './common/Button';
+import Popover from './common/Popover';
+
+// Logo Prices
+import { PRICES } from '../constants/data';
+
+/**
+ * PricePopover Component
+ *
+ * A popover component for filtering options.
+ * When the button is clicked, popover with a list of prices is displayed.
+ * Users can select specific options to apply filters, e.g., for a product list.
+ * The popover automatically closes when clicking outside of it.
+ *
+ * @returns {JSX.Element} The PricePopover Component
+ */
+const PricePopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const popoverRef = useRef(null);
+
+  // Handle clicks outside the popover to close it
+  const handleClickOutside = (event) => {
+    if (popoverRef.current && !popoverRef.current.contains(event.target)) {
+      setIsPopoverOpen(false);
+    }
+  };
+
+  useEffect(() => {
```

Comment:
duplicate here

Reply:
Yes, Fixed and updated by adding even handle useClickOutside into components duplicated even